react/lib/util/types.ts

```diff
@@ -15,6 +15,7 @@ export interface Transaction {
     timestamp: number
     address: string
     rawMessage?: string
+    inputAddresses?: (string | undefined)[]
```

Comment:
Same thing I commented in the server PR: I don't see the sense into having `undefined` as a possibility here.